frontend/packages/ceph-storage-plugin/integration-tests/views/pvc.view.ts

```diff
@@ -0,0 +1,89 @@
+import { $, browser, ExpectedConditions as until, element, by } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+
+export const selectItemFromDropdown = async (item, dropdownElement) => {
+  await browser.wait(until.presenceOf(dropdownElement));
+  await dropdownElement.click();
```

Comment:
```suggestion
  await click(dropdownElement);
  
```

Reply:
`click` inside `@console/shared/src/test-utils/utils` already takes care of it. Please change this at other places as well.

Reply:
Fixed, thank you.